newIDE/app/src/EventsBasedObjectEditor/index.js

```diff
@@ -69,22 +71,28 @@ export default function EventsBasedObjectEditor({ eventsBasedObject }: Props) {
         fullWidth
         rows={3}
       />
-      <I18n>
-        {({ i18n }) => (
-          <SemiControlledTextField
-            commitOnBlur
-            floatingLabelText={<Trans>Default name for created objects</Trans>}
-            value={
-              eventsBasedObject.getDefaultName() || eventsBasedObject.getName()
-            }
-            onChange={newName => {
-              eventsBasedObject.setDefaultName(gd.Project.getSafeName(newName));
-              forceUpdate();
-            }}
-            fullWidth
-          />
-        )}
-      </I18n>
+      <SemiControlledTextField
+        commitOnBlur
+        floatingLabelText={<Trans>Default name for created objects</Trans>}
+        value={
+          eventsBasedObject.getDefaultName() || eventsBasedObject.getName()
+        }
+        onChange={newName => {
+          eventsBasedObject.setDefaultName(gd.Project.getSafeName(newName));
+          forceUpdate();
+        }}
+        fullWidth
+      />
+      <Line>
```

Comment:
Weird that you need a line because you already use ColumnStackLayout which does the spacing between elements. See if you can remove it.

Reply:
ah, yes, It's because I dropped a fragment.